services/data_plane_manager/src/test/java/com/futurewei/alcor/dataplane/utils/DataPlaneManagerUtil.java

```diff
@@ -443,13 +441,6 @@ private void buildSubnetState(NetworkConfiguration networkConfig, Goalstate.Goal
             if (subnetEntity.getAvailabilityZone() != null) {
                 subnetConfigBuilder.setAvailabilityZone(subnetEntity.getAvailabilityZone());
             }
-            if (subnetEntity.getPrimaryDns() != null) {
-                subnetConfigBuilder.setPrimaryDns(subnetEntity.getPrimaryDns());
-            }
-
-            if (subnetEntity.getSecondaryDns() != null) {
-                subnetConfigBuilder.setSecondaryDns(subnetEntity.getSecondaryDns());
-            }
```

Comment:
Let us add //TODO to remind us we need to set DNS based on latest contract.

Reply:
done